core-common/src/main/java/org/glassfish/jersey/internal/inject/Providers.java

```diff
@@ -338,7 +339,7 @@ public RuntimeType getRuntime() {
                                                                 Annotation... qualifiers) {
 
         List<ServiceHolder<T>> serviceHolders = injectionManager.getAllServiceHolders(contract, qualifiers);
```

Comment:
Just for my OCD while you're here, could this method call the overloaded method directly above it?